src/components/Calendar/CalendarRoot.tsx

```diff
@@ -223,6 +224,18 @@ class ScheduleCalendar extends PureComponent<ScheduleCalendarProps, ScheduleCale
         canvas.style.color = 'black';
         calendarHeader.style.marginRight = '0px';
 
+        // Set each rbc header and time slot text color to white if the theme is dark
+        const rbc_headers = document.getElementsByClassName('rbc-header');
+        const rbc_time_slots = document.getElementsByClassName('rbc-label');
+
+        if (isDarkMode()) {
+            for (let i = 0; i < rbc_headers.length; i++) {
```

Comment:
Nitpick: can you change these to `for of` loops?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of